vmm/src/cpu.rs

```diff
@@ -1368,6 +1382,10 @@ impl Snapshottable for CpuManager {
             let vcpu = self
                 .create_vcpu(cpu_id.parse::<u8>().unwrap(), None, Some(*snapshot.clone()))
                 .map_err(|e| MigratableError::Restore(anyhow!("Could not create vCPU {:?}", e)))?;
+
+            // Adding vCPU to the CpuManager's vCPU list.
+            self.vcpus.push(vcpu.clone());
+
```

Comment:
How is this change related to the guest notification?

Reply:
Hehe it's not, that's why I'm gonna push it in its own commit :)

Reply:
done